tests/test_post_login.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import pytest
+
+def test_correct_login():
+    
+    response = requests.post('http://localhost:5000/login', json={"name" : "denchik", "password" : "foobar"})
+    assert response.status_code == 200
+    assert response.content == b'OK'
+
+def test_incorrect_login():
+
+    response = requests.post('http://localhost:5000/login', json={"name" : "den", "password" : "foobar"})
+    assert response.status_code == 403
+    assert response.content == b'FORBIDDEN'
+
+def test_invalid_header():
```

Comment:
It's better to use more descriptive test names, i.e. identify both the input and the result, for example, "json_without_content_type_header_returns_400"